apps/backend/src/modules/groups/group.controller.ts

```diff
@@ -216,10 +220,12 @@ class GroupController extends BaseController {
 	 *            schema:
 	 *              type: object
 	 *              properties:
-	 *                key:
-	 *                  type: string
 	 *                name:
 	 *                  type: string
+	 *                permissions:
+	 *                  type: array
+	 *                  items:
+	 *                    type: string
```

Comment:
```suggestion
	 *                    type: string
	                      minimum: 1
```
Add minimum length here, as we cannot create new group without permissions right now